sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -251,73 +247,610 @@ def update_queue(self, queue_description, **kwargs):
+        :rtype: ~azure.servicebus.management.TopicDescription
+        """
+        entry_ele = self._get_entity_element(topic_name, **kwargs)
+        entry = TopicDescriptionEntry.deserialize(entry_ele)
+        if not entry.content:
+            raise ResourceNotFoundError("Topic '{}' does not exist".format(topic_name))
+        topic_description = TopicDescription._from_internal_entity(entry.content.topic_description)
+        topic_description.name = topic_name
+        return topic_description
+
+    def get_topic_runtime_info(self, topic_name, **kwargs):
+        # type: (str, Any) -> TopicRuntimeInfo
+        """Get a TopicRuntimeInfo
+
+        :param str topic_name: The name of the topic.
+        :rtype: ~azure.servicebus.management.TopicRuntimeInfo
+        """
+        entry_ele = self._get_entity_element(topic_name, **kwargs)
+        entry = TopicDescriptionEntry.deserialize(entry_ele)
+        if not entry.content:
+            raise ResourceNotFoundError("Topic {} does not exist".format(topic_name))
+        topic_description = TopicRuntimeInfo._from_internal_entity(entry.content.topic_description)
+        topic_description.name = topic_name
+        return topic_description
+
+    def create_topic(self, topic, **kwargs):
+        # type: (Union[str, TopicDescription], Any) -> TopicDescription
+        """Create a topic.
+
+        :param Union[str, ~azure.servicebus.management.TopicDescription] topic: The topic name or a `TopicDescription`
+         instance. When it's a str, it will be the name of the created topic. Other properties of the created topic
+         will have default values decided by the ServiceBus.
+         Use a `TopicDescription` if you want to set queue properties other than the queue name.
+        :rtype: ~azure.servicebus.management.TopicDescription
+        """
+        try:
+            topic_name = topic.name  # type: ignore
+            to_create = topic._to_internal_entity()  # type: ignore  # pylint:disable=protected-access
+        except AttributeError:
+            topic_name = topic  # type: ignore
+            to_create = InternalTopicDescription()  # Use an empty topic description.
+
+        create_entity_body = CreateTopicBody(
+            content=CreateTopicBodyContent(
+                topic_description=to_create,  # type: ignore
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        with _handle_response_error():
+            entry_ele = cast(
+                ElementTree,
+                self._impl.entity.put(
+                    topic_name,  # type: ignore
+                    request_body, api_version=constants.API_VERSION, **kwargs)
+            )
+        entry = TopicDescriptionEntry.deserialize(entry_ele)
+        result = TopicDescription._from_internal_entity(entry.content.topic_description)
+        result.name = topic_name
+        return result
+
+    def update_topic(self, topic_description, **kwargs):
```

Comment:
I feel like this parameter should just be called `topic` to match the `create` API above.

Reply:
Updated.